shared/runtime/softtimer.h

```diff
@@ -30,6 +30,7 @@
 
 #define SOFT_TIMER_FLAG_PY_CALLBACK (1)
 #define SOFT_TIMER_FLAG_GC_ALLOCATED (2)
+#define SOFT_TIMER_FLAG_HARD_CALLBACK (4)
```

Comment:
Was 4 used here for a reason? Is 3 used elsewhere?

Reply:
These are bit masks that get ORed together, so powers of two.